app/modules/sightings/models.py

```diff
@@ -641,7 +641,14 @@ def get_matching_set_data(self, annotation_guid, matching_set_config=None):
                 log.warning(f'skipping {annot} due to no content_guid')
                 continue
             if annot.encounter and annot.encounter.sighting:
-                if annot.encounter.sighting.stage == SightingStage.processed:
+                log.debug(
+                    f'get_matching_set_data() at {annot.encounter.sighting} vs {self}'
+                )
+                # is not annot.encounter.sighting == self at this point????
```

Comment:
It absolutely should be so yes this should be a check on self and yes it should be allowed for identification and processed. Should it also be allowed for un-reviewed? In other words is it permitted for the user to rerun ID at the review stage?
If so then really the whole check on stage should be removed.

Reply:
ah yes, i was just thinking the same.  ok, will drop the whole check.

Reply:
actually, since `annot` comes from iterating over `matching_set_annotations`, i no longer believe the `self` thing.  however, i still think this test is too strict (maybe even more so given this fact), so am going to drop it.   in testing locally, it definitely unblocks some ident processing that i think should be going thru.

Reply:
(i also think the `if annot.encounter and annot.encounter.sighting...` might be redundant/unnecessary -- but am going to leave that to not rock the boat!  (i.e. i am pretty sure an annotation would _not_ get into `matching_set_annotations` if it was missing enc/sighting.)